locales/pt_BR/LC_MESSAGES/duckduckgo.po

```diff
@@ -4385,6 +4375,8 @@ msgid ""
 "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
 "Microsoft's ad network"
 msgstr ""
+"A privacidade é protegida pelo DuckDuckGo quando visualiza anúncios. Cliques "
+"em anúncios são gerenciados pela rede de anúncios da Microsoft."
```

Comment:
@marcellohalfeld This is also a contractually sensitive string and we cannot accept the translation. Could you please remove it?